inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp

```diff
@@ -1,70 +1,67 @@
-        std::tie(basicParamsSet, cpuParams) = obj.param;
+        fusingSpecificParams fusingParams;
+        std::tie(basicParamsSet, cpuParams, fusingParams) = obj.param;
 
         std::ostringstream result;
-        result << LayerTestsDefinitions::NormalizeL2LayerTest::getTestCaseName(testing::TestParamInfo<LayerTestsDefinitions::NormalizeL2LayerTestParams>(
-                basicParamsSet, 0));
-
+        result << NormalizeL2LayerTest::getTestCaseName(testing::TestParamInfo<NormalizeL2LayerTestParams>(basicParamsSet, 0));
         result << CPUTestsBase::getTestCaseName(cpuParams);
+        result << CpuTestWithFusing::getTestCaseName(fusingParams);
 
         return result.str();
     }
+
 protected:
     void SetUp() override {
-        LayerTestsDefinitions::NormalizeL2LayerTestParams basicParamsSet;
+        NormalizeL2LayerTestParams basicParamsSet;
         CPUSpecificParams cpuParams;
-        std::tie(basicParamsSet, cpuParams) = this->GetParam();
+        fusingSpecificParams fusingParams;
+        std::tie(basicParamsSet, cpuParams, fusingParams) = this->GetParam();
 
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+        std::tie(postOpMgrPtr, fusedOps) = fusingParams;
 
         std::vector<int64_t> axes;
         float eps;
-        ngraph::op::EpsMode eps_mode;
-        InferenceEngine::SizeVector inputShapes;
-        InferenceEngine::Precision  netPrecision;
+        op::EpsMode eps_mode;
+        SizeVector inputShapes;
+        Precision netPrecision;
         std::tie(axes, eps, eps_mode, inputShapes, netPrecision, targetDevice) = basicParamsSet;
+
         inPrc = outPrc = netPrecision;
         auto netPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
-        auto param = ngraph::builder::makeParams(netPrc, {inputShapes});
-        auto paramOuts = ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(param));
-        auto normalize_l2 = ngraph::builder::makeNormalizeL2(paramOuts[0], axes, eps, eps_mode);
-
-        ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(normalize_l2)};
+        auto params = builder::makeParams(netPrc, {inputShapes});
+        auto paramOuts = helpers::convert2OutputVector(helpers::castOps2Nodes<op::Parameter>(params));
+        auto normalize = builder::makeNormalizeL2(paramOuts[0], axes, eps, eps_mode);
 
-        if (Precision::BF16 == netPrecision) {
-            selectedType = "unknown_BF16";
-        } else if (Precision::FP32 == netPrecision) {
-            selectedType = "unknown_FP32";
-        }
+        function = makeNgraphFunction(netPrc, params, normalize, "Normalize");
 
+        selectedType = "unknown_" + std::string(netPrecision.name());
         threshold = 0.015f;
-
-        normalize_l2->get_rt_info() = getCPUInfo();
-
-        function = std::make_shared<ngraph::Function>(results, param, "Normalize");
+        posDepends = false;
```

Comment:
We for sure need more meaningful name: smt like `checkFusingPosition`. BTW why we shouldn't do it for this test?

Reply:
```ConvertNormalizeL2WithMulToNormalizeIE``` is ngraph transformation. It creates fusing names used ```FusedNames``` attributes, which is not depends from fused node position.